app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -76,13 +76,31 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
         arrayFormat: 'repeat'
       });
 
-      const tileParamsAsString = [regularParamsString, repeatParamsString]
+      // Need to use raw query params for bbox, because it's a mapbox template query param
+      const bboxString = bbox ? `bbox=${bbox}` : '';
```

Comment:
I don't think I understand the need for this. Why can't bbox be treated like other tileParams?

Reply:
it's a mapbox internal template/replacement token, so can't be urlencoded. see [here](https://docs.mapbox.com/style-spec/reference/sources/#tiled-sources)